nilearn/plotting/displays.py

```diff
@@ -137,6 +142,50 @@ def draw_left_right(self, size, bg_color, **kwargs):
                 bbox=dict(boxstyle="square,pad=0", ec=bg_color, fc=bg_color),
                 **kwargs)
 
+    def draw_scale_bar(self, bg_color, scale_width=5, units='cm', **kwargs):
+        """ Adds a scale bar annotation to the display
+
+        Parameters
+        ----------
+
+        bgcolor: matplotlib color: string or (r, g, b) value
+            The background color of the scale bar annotation
+        scale_width: float (default: 5.0)
+            A width for the scale given in ``units``
+        units: str
+            Physical units to use on the scale bar (`'cm'` or `'mm'`)
+        kwargs:
+            Extra keyword arguments are passed to matplotlib's text
+            function.
+        """
+        from mpl_toolkits.axes_grid1.anchored_artists import AnchoredSizeBar
+        axis = self.ax
+        font_properties = kwargs.get('fontproperties', FontProperties())
+        if kwargs.get('size'):
+            font_properties.set_size(kwargs.pop('size'))
+
+        width_mm = scale_width or kwargs.pop('scale_width', 5)
+        if units == 'cm':
+            width_mm *= 10
+
+        frame_on = kwargs.pop('frameon', False) is True
+        anchor_size_bar = AnchoredSizeBar(
+            axis.transData,
+            width_mm,
+            '%g%s' % (scale_width, units),
+            loc=kwargs.pop('loc', 4),
+            pad=kwargs.pop('pad', 0.1),
+            borderpad=kwargs.pop('borderpad', 0.5),
+            sep=kwargs.pop('sep', 5),
```

Comment:
These arguments should not be in the kwargs, but as explicit arguments to the function, with the documentation, as they are fairly central to its use.

Also, the docstring should document where the kwargs are sent, so that people can look up what kwargs are acceptable.